src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -51,7 +54,19 @@ const imageWidthSizes = {
   xl: 1112,
 }
 
+@track({
+  context_module: Schema.ContextModule.CollectionDescription,
+})
 export class CollectionHeader extends Component<Props> {
+  @track({
+    subject: Schema.Subject.ReadMore,
+    type: Schema.Type.Button,
+    action_type: Schema.ActionType.Click,
+  })
+  trackReadMoreClick() {
+    // noop
+  }
```

Comment:
cc @yuki24 - moved tracking directly into the header and updated to follow new schema patterns